Python/Bollywood.py

```diff
@@ -0,0 +1,43 @@
+import random
+movies = ('LAGAAN', 'HIGHWAY', 'THOR', 'DANGAL', 'NEWTON','ANEK','APNE','CHHICHHORE','WAR','GULAAL','DHOOM','MALANG', 'STAR WARS')
+c_m = random.sample(movies,1)
+k = '*' * len(c_m[0])
```

Comment:
- instead of i,j,k c_m use proper variable naming conventions. Refer this link https://www.python.org/dev/peps/pep-0008/

- Give proper spacing between lines

Reply:
done sir